db/seeds.rb

```diff
@@ -37,3 +37,77 @@
     AlaveteliPro::RequestSummaryCategory.create(slug: category_slug)
   end
 end
+
+OutgoingMessageSnippet.create(
```

Comment:
Be good to add ~50+ snippets – even if they're lorem ipsum – just so we can see how it feels with several snippets appearing under a given section.